quickfixj-core/src/main/doc/usermanual/usage/configuration.html

```diff
@@ -1285,11 +1285,21 @@ <H3>QuickFIX Settings</H3>
   <TR ALIGN="left" VALIGN="middle">
     <TD><I>TestRequestDelayMultiplier</I></TD>
     <TD>Fraction of the heartbeat interval which defines the additional time to wait
-    if a TestRequest sent after a missing heartbeat times out.
+    if a TestRequest sent after a missing heartbeat times out (final coefficient value is equal to
```

Comment:
I don't know if I made it worst by clarifying the final coefficient values as for TestRequestDelayMultiplier there is also number of requests variable which might require to look in in the code anyway.